lib/auth/auth_with_roles.go

```diff
@@ -3339,3 +3329,7 @@ func verbsToReplaceResourceWithOrigin(stored types.ResourceWithOrigin) []string
 	}
 	return verbs
 }
+
+// metaKindClusterConfig is used when checking access to cluster configuration
+// resources, as an alternative to their individual resource kinds.
+const metaKindClusterConfig = "cluster_config"
```

Comment:
TBH, I would've just kept `types.KindClusterConfig` and just mark as deprecated and used only for backwards compatibility via a godoc.